static/images/LICENSE

```diff
@@ -0,0 +1,18 @@
+Images in this directory are:
+(c) Map Tiler  (c) OpenStreetMap contributors
+
+See: https://www.openstreetmap.org/copyright
+OpenStreetMap is open data, licensed under the Open Data Commons Open Database License (ODbL) by the OpenStreetMap Foundation (OSMF).
+OpenStreetMap is a registered trademark of the OpenStreetMap Foundation.
+
+See https://www.maptiler.com/copyright/
+
+Open Data Commons Open Database License v1.0
+--------------------------------------------------
+
+PREAMBLE
+The Open Database License (ODbL) is a license agreement intended to allow users to freely share, modify, and use this
+
+...fill in from https://opendatacommons.org/licenses/odbl/1-0/... down to
```

Comment:
I meant to paste in the text from the link please - it was quite long to insert in the jira ticket.